gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+networkTimeout=10000
```

Comment:
Is this something you added, when I ran `./gradlew wrapper --gradle-version=7.6` in another project this was not added.

Reply:
It was added for me, IIRC in all projects I ran it in, not just here. It was added in 7.6 according to the gradle repo too.